CV_Ilya_Siamionau.md

```diff
@@ -31,41 +31,41 @@ css: |-
 
 <h2 class="my-3">SUMMARY</h2>
 
-<p style="text-indent: 25px;">Software Engineer with 2.5+ years of strong experience in Backend Development in Python and JavaScript, practical experience in real-time communication (WebRTC) and Front-end Development in React + Redux, first-hand expertise in making C++ bindings for Python. Vocational education taken after high education level.</p>
+<p style="text-indent: 25px;">Software engineer with 2.5+ years of strong experience in back-end development in Python and JavaScript, practical experience in real-time communication (WebRTC) and front-end development in React + Redux, first-hand expertise in making C++ bindings for Python. Vocational education taken after high education level.</p>
 
 <h2 class="my-3">EXPERIENCE</h2>
 
 <div class="row d-flex justify-content-between align-items-center">
   <div class="col text-start">
     <a href="https://abelanavr.com/" class="fs-6">Abelana VR</a>
-    <span>as a backend developer</span>
+    <span>as a back-end developer</span>
```

Comment:
Its personality. Any can write it how wish. Ref: https://ell.stackexchange.com/questions/117383/what-is-the-correct-term-back-end-back-end-or-backend

Reply:
I did put "back-end" here because it is a compound adjective. [\[1\]](https://www.grammarly.com/blog/hyphens-and-dashes/)